src/__tests__/get-by-errors.js

```diff
@@ -97,6 +97,33 @@ cases(
   },
 )
 
+test.each([['getByText'], ['getByLabelText']])(
+  '%s query will print the playground link when enabled in the config',
+  query => {
+    configure({printPlaygroundLink: true})
+    document.body.innerHTML = '<div>Hello</div>'
+    expect(() => screen[query]('TEST QUERY')).toThrowErrorMatchingSnapshot()
+  },
+)
+
+test.each([['getByText'], ['getByLabelText']])(
+  '%s query will NOT print the playground link when disabled in the config',
+  query => {
+    configure({printPlaygroundLink: false})
```

Comment:
Since these are side-effects, we need to make sure they're cleaned up after the test. So something like
1. save current config in `beforeEach`
2. set saved config in `afterEach`

Reply:
Something like that is already in place in this file :-)

https://github.com/testing-library/dom-testing-library/blob/5bc93643f312d9ca4210b97681686c9aa8a902d7/src/__tests__/get-by-errors.js#L6-L9

Reply:
I guess changing it to an `afterEach` will make sure it's also cleaned up after the last test runs inside this file.

Reply:
I have changed it to `afterEach` :-)
https://github.com/testing-library/dom-testing-library/pull/852/commits/1c4f22b754e1b3d52539d762f7109ba55290bd7b